config/channels/in-memory-channel/resources/in-memory-channel.yaml

```
@@ -74,6 +74,9 @@ spec:
                       CACerts:
                         description: Certification Authority (CA) certificates in PEM format that the source trusts when sending events to the sink.
                         type: string
+                      audience:
+                        description: Audience is the OIDC audience. This need only be set, if the target is not an Addressable and thus the Audience can't be received from the Addressable itself. In case the target is an Addressable and specifies an Audience too, the Destinations Audience takes preference.
```

Nit: I think this wording is clearer
```suggestion
                        description: Audience is the OIDC audience. This only needs to be set if the target is not an Addressable and thus the Audience can't be received from the Addressable itself. If the target is an Addressable and specifies an Audience, the target's Audience takes precedence.
```

---

@Cali0707 @xiangpingjiang

---

Ah sorry my bad @mgencur! Thanks for catching + opening a fix